Dockerfile.base

```diff
@@ -101,12 +101,18 @@ RUN apt update \
 		xfonts-base \
 		xfonts-cyrillic \
 		xfonts-scalable \
-		xvfb
+		xvfb \
+		sudo
 
 RUN wget --no-verbose https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}_amd64.deb \
 	&& apt-get install -y ./google-chrome-stable_${CHROME_VERSION}_amd64.deb \
 	&& rm ./google-chrome-stable_${CHROME_VERSION}_amd64.deb
 
+# Add user calypso with 1002, give it sudo permissions
+RUN adduser --uid 1002 --disabled-password calypso \
```

Comment:
Would it make sense to add this to the base image? Do we need it for other things?

Reply:
Good idea, moved to the base image.